targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F469xI/TARGET_SDP_K1/PinNames.h

```diff
@@ -309,21 +309,14 @@ typedef enum {
     LED1        = PK_7, // Red LED
     LED2        = PK_6, // Orange LED
     LED3        = PK_5, // Green LED
+    LED4        = PK_4,
     LED_RED     = LED1,
     LED_ORANGE  = LED2,
     LED_GREEN   = LED3,
     SERIAL_TX   = STDIO_UART_TX,
     SERIAL_RX   = STDIO_UART_RX,
     USBTX       = STDIO_UART_TX,
     USBRX       = STDIO_UART_RX,
-    
-    // SPI and I2C pins on Arduino connector
-    SPI_CS      = D10,
-    SPI_MOSI    = D11,
-    SPI_MISO    = D12,
-    SPI_SCK     = D13,
-    I2C_SDA     = D14,
-    I2C_SCL     = D15,
```

Comment:
Why is this being removed? Dxx can still be used, this is just having SPI/I2C pins as well .

Reply:
This change was intended and the lines are being removed so that only the Arduino connector pin names (D0-D15) are used in example programs for this board. This is to ensure code portability to other Mbed enabled boards with Arduino form factor.
@0xc0170

Reply:
Hello, can this code be merged? Is there a reason to keep those lines of code, though we have reasons to have them removed?
@0xc0170 @adbridge @bentcooke

Reply:
This should be OK, it will work - any program using Dxx will function. I recall we have this SPI_/I2C_ pin names defined for almost every board, not certain how this would be different

@MarceloSalazar @jamesbeyond Can you review?

Reply:
Better compatibility would be with these lines left in here

Reply:
@0xc0170 That commit has been removed. Please merge the file. We need at least rest of the changes to be released. 
The pin names issue can be resolved later.